Formula/gitlab-ci-local.rb

```diff
@@ -69,10 +71,7 @@ def install
         merge = refs/heads/master
     EOS
 
-    assert_equal shell_output("#{bin}/gitlab-ci-local --list"), <<~OUTPUT
-      name              description  stage  when        allow_failure  needs
-      build                          build  on_success  false          []
-      tag-docker-image               tag    on_success  false          [build]
-    OUTPUT
+    assert_match(/name              description  stage  when        allow_failure  needs\n/,
```

Comment:
This is probably dependent on terminal width, so let's just allow any whitespace
```suggestion
    assert_match(/name\s*?description\s*?stage\s*?when\s*?allow_failure\s*?needs\n/,
```

Reply:
👍